### PR TITLE
add allChildren and getNextPage functions

### PR DESCRIPTION
**Pull request by jchen293**
# Description

- Adds `allChildren` function in User service to get a paginated list of children
- Adds `getNextPage` function in User service to get next paginated list of children
# Testing

Add unit tests
# Pull Request Type

Please select the option(s) that are relevant to this PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement (fixing a typo, updating readme, renaming a variable name, etc)

### CODE REVIEW

File: src/main/java/com/easypost/model/Children.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.easypost.model;
+
+import java.util.List;
+
+public class Children extends EasyPostResource {
+    private String parentId;
+    private String name;
+    private String phoneNumber;
+    private Boolean verified;
+    private Boolean defaultCarbonOffset;
```
Comment (Justintime50) [inline src/main/java/com/easypost/model/Children.java:10]: question: is this still valid? I thought this key was gone

File: src/main/java/com/easypost/service/UserService.java [hunk 2]
```diff
@@ -137,4 +142,34 @@ public Brand updateBrand(final String id, final Map<String, Object> params) thro
...
+     * @throws EasyPostException when the request fails.
+     */
+    public ChildrenCollection allChildren(final Map<String, Object> params) throws EasyPostException {
+        String endpoint = "users/children";
+        
+        return Requestor.request(RequestMethod.GET, endpoint, params, ChildrenCollection.class, client);
+    }
+
+    /**
+     * Get the next page of an ChildrenCollection.
```
Comment (Justintime50) [inline src/main/java/com/easypost/service/UserService.java:160]:
```suggestion
     * Get the next page of a ChildrenCollection.
```

File: src/main/java/com/easypost/service/UserService.java [hunk 3]
```diff
@@ -137,4 +142,34 @@ public Brand updateBrand(final String id, final Map<String, Object> params) thro
 
         return Requestor.request(RequestMethod.PUT, endpoint, wrappedParams, Brand.class, client);
     }
+
+    /**
+     * Retrieve the paginated list of children of a user.
```
Comment (Justintime50) [inline src/main/java/com/easypost/service/UserService.java:147]:
```suggestion
     * Retrieve the paginated list of children for the authenticated user.
```

**Review by Justintime50 — DISMISSED** [review]

File: src/main/java/com/easypost/model/Children.java (hunk 1 above)
Comment (jchen293) [reply src/main/java/com/easypost/model/Children.java:10]: Good catch, I will remove this

**Review by nwithan8 — APPROVED** [review]

**Review by nwithan8 — APPROVED** [review]

**Review by nwithan8 — APPROVED** [review]

**Pull request merged**